packages/circuit-ui/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { HTMLAttributes } from 'react';
+import { css } from '@emotion/core';
+import isPropValid from '@emotion/is-prop-valid';
+
+import styled, { StyleProps } from '../../styles/styled';
+
+export interface AvatarProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, 'size'> {
+  /**
+   * Image to render
+   */
+  imageUrl?: string;
+  /**
+   * Shape of the Avatar
+   */
+  variant?: 'square' | 'round';
+  /**
+   * Size of the Avatar
+   */
+  size?: 'small' | 'large';
```

Comment:
The size names should be consistent with other components. These are the ones Giao-Chung used in Figma:

```suggestion
  size?: 'giga' | 'yotta';
```